sentences/en/_common.yaml

```diff
@@ -48,7 +48,7 @@ expansion_rules:
   area: "[the] {area}"
   what_is: "(what's | whats | what is)"
   brightness: "{brightness} [percent]"
-  turn: "(turn | switch | change)"
+  turn: "(turn | switch | change | shut)"
```

Comment:
"shut" is only used for shutting something off, not on. Maybe it's better to add it in the HassTurnOff intent (e.g. `(<turn> | shut) off ...`).

Reply:
That's a good point, I agree that makes more sense.

Reply:
Updated